includes/admin/views/builder/question.php

```diff
@@ -1,9 +1,11 @@
 <?php
 /**
- * Builder question view
+ * Builder question view.
  *
- * @since   3.16.0
- * @version 3.27.0
+ * @since 3.16.0
+ * @since [version] Added checkbox for including question in Question Bank.
+ *
+ * @version [version]
```

Comment:
```suggestion
 * @since 3.16.0
 * @since 3.27.0 Unknown.
 * @since [version] Added checkbox for including question in Question Bank.
 * @version [version]
```